src/settings.h

```diff
@@ -8,8 +8,22 @@
 #include "wayland/protocols/wlr-layer-shell-unstable-v1-client-header.h"
 #endif
 
-#include "notification.h"
+#ifdef ENABLE_X11
 #include "x11/x.h"
+#endif
+
+// Note: Wayland doesn't support hotkeys
```

Comment:
Maybe a bit strict, but shouldn't the comment be that `global hotkeys` aren't possible?
Hotkeys for active windows in general exist, but those don't work well with how dunst operates

Reply:
you are totally right. I am not a great expert of wayland myself, so I was not sure what to write. Initially I thought that dunst didn't implement them, but then I saw that by design wayland didn't allow them (for dunst)